### PR TITLE
Fix the db prefix not being removed from request parameters in method refresh_privileges

### PR DESCRIPTION
**Pull request by flaming-archer**
### :pencil: Description
Fix the db prefix not being removed from request parameters.
This issue will cause the log to keep reporting errors.

`16:32:57.906 [pool-6-thread-73] INFO  com.hotels.bdp.waggledance.client.compatibility.HiveCompatibleThriftHiveMetastoreIfaceFactory - Couldn't invoke method public abstract org.apache.hadoop.hive.metastore.api.GrantRevokePrivilegeResponse org.apache.hadoop.hive.metastore.api.ThriftHiveMetastore$Iface.refresh_privileges(org.apache.hadoop.hive.metastore.api.HiveObjectRef,java.lang.String,org.apache.hadoop.hive.metastore.api.GrantRevokePrivilegeRequest) throws org.apache.hadoop.hive.metastore.api.MetaException,org.apache.thrift.TException`

### :link: Related Issues

### CODE REVIEW

File: waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java [hunk 1]
```diff
@@ -1341,6 +1341,14 @@ public GrantRevokePrivilegeResponse grant_revoke_privileges(GrantRevokePrivilege
   public GrantRevokePrivilegeResponse refresh_privileges(HiveObjectRef hiveObjectRef, String authorizer,
                                                          GrantRevokePrivilegeRequest grantRevokePrivilegeRequest) throws MetaException, TException {
     DatabaseMapping databaseMapping = checkWritePermissions(hiveObjectRef.getDbName());
+    PrivilegeBag privilegeBag = grantRevokePrivilegeRequest.getPrivileges();
```
Comment (patduin) [inline waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java:1344]: could you update the Unit test as well please?

Comment (flaming-archer) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java]: Updated, pls review.

**Comment by flaming-archer** [comment]
This failed test case `get_table_meta` seems to have been consistently slow and prone to timeout.  It shouldn't have been affected by my changes

**Review by patduin — APPROVED** [review]

**Comment by patduin** [comment]
Thank you @flaming-archer!

**Pull request merged**